ci/pipelines/default-pipeline.yml

```diff
@@ -113,6 +113,7 @@ analysis:detekt-custom:
     - ./gradlew :features:dd-sdk-android-webview:customDetektRules
     - ./gradlew :integrations:dd-sdk-android-coil:customDetektRules
     - ./gradlew :integrations:dd-sdk-android-compose:customDetektRules
+    - ./gradlew :integrations:dd-sdk-android-navigation3:customDetektRules
```

Comment:
Is Navigation 3 only for Compose? Maybe we should call this module `dd-sdk-android-compose-navigation3` to be more precise? WDYT?